src/iuse.cpp

```diff
@@ -4899,6 +4899,16 @@ int iuse::artifact( player *p, item *it, bool, const tripoint & )
                 }
             }
             break;
+            
```

Comment:
A couple of lines with astyle regression here and below:

![image](https://user-images.githubusercontent.com/16213433/42129533-a369b3cc-7ccf-11e8-80ca-d49c54b53cb8.png)

Reply:
Whoops, should be fixed now.